src/humancellatlas/data/metadata/api.py

```diff
@@ -233,12 +233,17 @@ def __init__(self, entity: LinkedEntity, other_entity: Entity, forward: bool) ->
 class ProjectPublication:
     title: str
     url: Optional[str]
+    official_hca_publication: Optional[bool]
```

Comment:
```suggestion
    official_hca: Optional[bool]
```

since we stripped that word from the other property names as well.